grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2364,7 +2366,7 @@ class StockMovementService {
     }
 
     void createMissingShipmentItems(StockMovement stockMovement) {
-        Requisition requisition = stockMovement.requisition.refresh()
+        Requisition requisition = Requisition.get(stockMovement.id)
```

Comment:
I assume we're getting this requisition from the session so we should be getting the same object. Curious why we used refresh in the first place (https://pihemr.atlassian.net/browse/OBPIH-1257). 

As such, I wonder if this
```
Requisition requisition = stockMovement.requisition
```
would suffice. 

One worry I have is with stockMovement.id which could potentially represent another object like order, shipment, etc. Would that ever be the case? And if not, we should be explicit about handling these cases i.e. throw an exception.

Reply:
But that could be done as a post-migration refactoring. 

I'm not a big fan of these methods, in general. For one, they are too generically named and should be part of a more specific to the work we're trying to accomplish i.e. 
```
void refreshPicklist(RequisitionItem requisitionItem)
```

Reply:
I checked locally the `stockMovement.requisition` and looks like I had weird glitch or whatever, because all versions:
```
stockmovement.requisition
stockmovement.requisition.refresh()
Requisition.get(stockmovement.id)
```
now all return the same state of requisitionItems. :thinking: 
I'll change it to the `stockmovement.requisition` and push within my next PR.